context/app/api/vitessce_confs/base_confs.py

```diff
@@ -177,39 +177,33 @@ def _setup_scatterplot_view_config(self, vc, dataset):
 
 class SPRMViewConfBuilder(ImagePyramidViewConfBuilder):
 
-    def _get_sprm_image(self):
-        image_file_regex = f"{self._imaging_path}/{self._base_name}.ome.tiff?"
+    def _make_full_image_path(self):
+        return f"{self._imaging_path}/{self._image_name}.ome.tiff?"
+
+    def _check_sprm_image(self, path_regex):
         file_paths_found = self._get_file_paths()
-        found_image_files = get_matches(file_paths_found, image_file_regex)
+        found_image_files = get_matches(file_paths_found, path_regex)
         if len(found_image_files) != 1:
             message = f'Found {len(found_image_files)} image files for SPRM uuid "{self._uuid}".'
             raise FileNotFoundError(message)
         found_image_file = found_image_files[0]
         return found_image_file
 
-    def _add_sprm_image_to_view_conf(self, found_image_file, vc, dataset):
+    def _get_sprm_ometiff_wrapper(self, found_image_file):
         img_url, offsets_url = self._get_img_and_offset_url(
             found_image_file, self._imaging_path,
         )
-        image_wrapper = OmeTiffWrapper(
+        return OmeTiffWrapper(
             img_url=img_url, offsets_url=offsets_url, name=self._base_name
         )
-        dataset = dataset.add_object(image_wrapper)
-        return dataset
-
-    def _perpare_vc_and_dataset_with_image(self):
-        found_image_file = self._get_sprm_image()
-        vc = VitessceConfig(name=self._base_name)
-        dataset = vc.add_dataset(name="SPRM")
-        dataset = self._add_sprm_image_to_view_conf(found_image_file, vc, dataset)
-        return vc, dataset
 
 
 class SPRMJSONViewConfBuilder(SPRMViewConfBuilder):
     def __init__(self, entity, nexus_token, is_mock=False, **kwargs):
         # All "file" Vitessce objects that do not have wrappers.
         super().__init__(entity, nexus_token, is_mock)
         self._base_name = kwargs["base_name"]
+        self._image_name = kwargs["base_name"]
```

Comment:
This is correct? Assigning the same value to two separate variables is a little odd... perhaps add a note to make it clear that this is right?

Reply:
Yes, unlike with the stitched version there is no `_stitched_expressions` to worry about